spark/src/main/java/org/opensearch/sql/spark/flint/operation/FlintIndexOpCancel.java

```diff
@@ -20,57 +19,46 @@ public class FlintIndexOpCancel extends FlintIndexOp {
   private static final Logger LOG = LogManager.getLogger();
 
   private final EMRServerlessClient emrServerlessClient;
+  private final boolean isDropOperation;
```

Comment:
any special reason to merge Drop and Cancel together if they may run individually? I think the previous design follows single responsibility principle?

Reply:
Cancel operation is not an actual operation --> which is ideally alter and we already have a separate operation.

In case after discussions with peng, I better understand the cancel use case. I will have a separate class for it.